### PR TITLE
Optimize installation experience on windows

### PR DESCRIPTION
**Pull request by zhu0629**

### CODE REVIEW

File: lib/util.lua [hunk 1]
```diff
@@ -65,27 +67,98 @@ function windowsCompile(ctx)
...
-            local contents = logFile:read("*a")
-            logFile:close()
-            print("=====================================")
-            print(qInstallLog)
-            print(contents)
-            print("=====================================")
-            print("Please uninstall python manually and try again.")
+    -- Extract
+    print("Extracting installer...")
+    local wixBin = os.getenv("USERPROFILE") .. '\\.version-fox\\plugin\\python\\bin\\WiX\\dark.exe'
```
Comment (aooohan) [inline lib/util.lua:72]: You can use `RUNTIME.pluginDirPath` to get the path of plugin home.

https://github.com/version-fox/vfox-plugin-template/blob/fddd1526732f6e2552c2725a63d4d56b5de34cc0/Injection.lua#L15

**Review by aooohan — CHANGES_REQUESTED** [review]
Great job! Thank you so much for such a great contribution! 👍👍👍

I will test it on Windows later.

File: lib/util.lua (hunk 1 above)
Comment (zhu0629) [reply lib/util.lua]: resolved.

**Review by aooohan — APPROVED** [review]
LGTM.

Test passed!

**Pull request merged**